packages/vaadin-grid-pro/test/edit-column-template.test.js

```diff
@@ -156,15 +158,16 @@ describe('edit column template', () => {
       expect(column.editorType).to.be.equal('custom');
     });
 
-    it('should reset the column `editorType` to text when template is removed', () => {
-      column._editModeTemplate = null;
-      expect(column.editorType).to.be.equal('text');
-    });
+    // it('should reset the column `editorType` to text when template is removed', () => {
```

Comment:
As we remove the Template API from `vaadin-grid-pro`, this test is no longer relevant.